llama_hub/library.json

```diff
@@ -39,6 +39,10 @@
     "id": "assemblyai",
     "author": "patrickloeber"
   },
+  "AstraDBReader": {
+    "id": "astra_db_reader",
```

Comment:
I think i found the reason why tests are failing: `id` here should be the same as the directory name which in this case is `astra_db`

Reply:
Aha, my mistake!! yes i see that now. I'll push the update shortly. thank you!